run/types/DeviceWrapper.ts

```diff
@@ -809,14 +863,32 @@ export class DeviceWrapper implements SharedDeviceInterface {
     return el;
   }
 
+  public async waitForLoadingAnimation() {
+    let loadingAnimation: AppiumNextElementType | null = null;
+
+    do {
+      try {
+        loadingAnimation = await this.waitForTextElementToBePresent({
+          strategy: "id",
+          selector: "network.loki.messenger:id/thumbnail_load_indicator",
+        });
+        await sleepFor(100);
+        console.info("loading-animation was found, waiting for it to be gone");
+      } catch (e: any) {
+        console.log("Loading animation not found");
+      }
+    } while (loadingAnimation);
+    console.info("Loading animation has finished");
+  }
```

Comment:
this is android only I suppose? Maybe mark it as such?

Reply:
or throw if the device is not an android one for now.

Reply:
I am also not using this function anywhere...